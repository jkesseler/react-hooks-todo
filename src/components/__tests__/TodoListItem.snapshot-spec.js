import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import TodoListItem from '../TodoListItem';

describe('TodoListItem', () => {
  it('renders without crashing', () => {
    const component = shallow(<TodoListItem />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
