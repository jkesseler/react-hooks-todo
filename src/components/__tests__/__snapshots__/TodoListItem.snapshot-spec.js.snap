// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListItem renders without crashing 1`] = `
<WithStyles(ForwardRef(ListItem))
  divider={false}
>
  <WithStyles(ForwardRef(Checkbox))
    checked={false}
    disableRipple={true}
  />
  <WithStyles(ForwardRef(ListItemText)) />
  <WithStyles(ForwardRef(ListItemSecondaryAction))>
    <WithStyles(ForwardRef(IconButton))
      aria-label="Delete todo"
    >
      <DeleteOutlinedIcon />
    </WithStyles(ForwardRef(IconButton))>
  </WithStyles(ForwardRef(ListItemSecondaryAction))>
</WithStyles(ForwardRef(ListItem))>
`;
